scripts/pcvGuardianDeploy.ts

```diff
@@ -0,0 +1,77 @@
+import hre, { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { utils } from 'ethers';
+import { keccak256 } from 'ethers/lib/utils';
+import config from './config';
+import { Core, PCVGuardian } from '@custom-types/contracts';
+
+const { CORE, PROTOCOL_MULTISIG_ADDRESS, VOLT_FUSE_PCV_DEPOSIT, PCV_DEPOSIT, PRICE_BOUND_PSM } = config;
+
+const PCV_GUARD_ROLE = keccak256(utils.toUtf8Bytes('PCV_GUARD_ROLE'));
+const PCV_GUARD_ADMIN_ROLE = keccak256(utils.toUtf8Bytes('PCV_GUARD_ADMIN_ROLE'));
+
+const pcvGuardAddress1 = '0xf8D0387538E8e03F3B4394dA89f221D7565a28Ee';
```

Comment:
What are these addresses? Hard to audit what you're granting the pcvGuard role to like this. Can you name and pull from the config? Do you have a `mainnetAddresses.ts` or equivalent?

Reply:
Yeah I would add them to a config as `GUARD_EOA_1` and `GUARD_EOA_2` or similar